packages/js/src/ui/components/Notification/NotificationList.tsx

```diff
@@ -17,7 +17,7 @@ const EmptyNotificationList = () => {
     <div
       class={style(
         'notificationListEmptyNoticeContainer',
-        'nt-absolute nt-inset-0 nt-flex nt-flex-col nt-items-center nt-m-auto nt-h-fit nt-w-full nt-text-foreground-alpha-100'
+        'nt-flex nt-flex-col nt-items-center nt-m-auto nt-h-fit nt-w-full nt-text-foreground-alpha-100 nt-p-6 nt-text-center'
```

Comment:
Adding padding to the empty state, to ensure custom localization prop has appropriate padding. The added padding is equivalent to the padding around other parts of the view (p-6)

<img width="360" alt="image" src="https://github.com/user-attachments/assets/b5792d12-effe-4dd9-9cb1-3cdd9d51c76a">